_episodes/06-agile-in-research.md

```diff
@@ -0,0 +1,138 @@
+> > Emergent design in iterative sprints does not always lead to the best large-scale functionality.
+> > ### Team dedication challenge
+> > Agile is often explained with the idea of a team fully dedicated to a single project.
+> > In academia team members are often involved in numerous projects in and outside of the team.
+> > ### Product owner challenge
+> > For a team running multiple research projects, who acts as product owner?
+> > ### Distributed expertise challenge
+> > In Agile methodologies it is often assumed that a task can be picked up by any team member.
+> > However, in science different team members often have specialized in certain domains or technologies,
+> > and it makes more sense to assign specific tasks to members with a specific expertise.
+> > ### What is value challenge
+> > The goal of Agile way of working is to maximize value for the customer,
+> > but what is value in research? And who is the customer?
+> {: .solution}
+{: .discussion}
+-----------------------------------------------
+
+> ## Discussion
+> Can you think of solutions to these challenges?
+{: .discussion}
+
+### Uncertainty Challenge
+When doing science, the time required to implement a solution is ambiguous and depends on unknowns such as
+outcomes of investigations and experiments or data availability.
+#### Solutions:
+- **Use *spikes***: The team could create *spikes*: items in the backlog that
+provide time for research and discovery.
+They are either timeboxed or their definition of done is when the specified research objectives are met.
+- **Do the least understood work first**: Start with the most uncertain tasks and increment from there.
+- **Deviate from the plan**: The sprint plan is not set in stone,
+teams should 'negotiate the scope of the Sprint Backlog' when needed.
+- **Relax "Definition of Done"**: To keep a fast discovery pace,
+exploratory work doesn't need to meet the same strict criteria as regular tasks.
+
+----------------------------------------------------
+### Cultural challenge
+A scientist is traditionally used to working individually in an isolated silo.
+Also management might be hesitant to let teams self-manage.
+Given this individualistic culture it is often intimidating to start with Agile practices.
+#### Solutions:
+- **Implement flexibly**: You should fit the methodology to the way of working.
+'Process-rigidity is the antithesis of agility.'
+- **Emphasize the "why"**: Focus on the "why" for the change to agile instead of
+"what" to do to make the change.
+
+----------------------------------------------------
+### Waterfall project setup challenge
+Scientific projects are often setup in long-term project timelines following a
+Waterfall method.
+#### Solutions:
+- Clear communication is key. Inform your collaborators on your way of working and update them frequently.
+- Actually projects that follow the Waterfall method often do not make the targets set out at the start of the project.
```

Comment:
As written this doesn't really seem like a "solution". Also, in many big projects that use waterfall-like project management then failure to meet the targets can mean not being paid.